tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -353,7 +353,7 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
 
             TraceId128BitGenerationEnabled = config
                                             .WithKeys(ConfigurationKeys.FeatureFlags.TraceId128BitGenerationEnabled)
-                                            .AsBool(false);
+                                            .AsBool(true);
```

Comment:
I assume we don't have any back-compat concerns with this right?

Reply:
This is what I understood. We can anyway bring @zacharycmontoya or @lucaspimentel in the loop to double confirm.

Reply:
Correct, there are no back-compat concerns according to our OpenTelemetry team